airflow/config_templates/default_celery.py

```diff
@@ -23,8 +23,10 @@
     'accept_content': ['json', 'pickle'],
     'event_serializer': 'json',
     'result_serializer': 'pickle',
+    'task_reject_on_worker_lost': True,
     'worker_prefetch_multiplier': 1,
     'task_acks_late': True,
+    'task_acks_on_failure_or_timeout': False,
```

Comment:
is `acks` right? it may be, just wanna check it isn't a typo!

Reply:
It is :) :
https://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-task_acks_on_failure_or_timeout

Reply:
nice, acknowledge ~ ack. good to know! lol